gui.go

```diff
@@ -620,17 +703,54 @@ func (g *Gui) onKey(ev *termbox.Event) error {
 // execKeybindings executes the keybinding handlers that match the passed view
 // and event. The value of matched is true if there is a match and no errors.
 func (g *Gui) execKeybindings(v *View, ev *termbox.Event) (matched bool, err error) {
-	matched = false
+	var globalKb *keybinding
 	for _, kb := range g.keybindings {
 		if kb.handler == nil {
 			continue
 		}
-		if kb.matchKeypress(Key(ev.Key), ev.Ch, Modifier(ev.Mod)) && kb.matchView(v) {
-			if err := kb.handler(g, v); err != nil {
-				return false, err
-			}
-			matched = true
+		if !kb.matchKeypress(Key(ev.Key), ev.Ch, Modifier(ev.Mod)) {
+			continue
+		}
+		if kb.matchView(v) {
+			return g.execKeybinding(v, kb)
 		}
+		if kb.viewName == "" && (!v.Editable || kb.ch == 0) {
+			globalKb = kb
+		}
+	}
+	if globalKb != nil {
+		return g.execKeybinding(v, globalKb)
+	}
+	return false, nil
+}
+
+// execKeybinding executes a given keybinding
+func (g *Gui) execKeybinding(v *View, kb *keybinding) (bool, error) {
+	if err := kb.handler(g, v); err != nil {
+		return false, err
 	}
-	return matched, nil
+	return true, nil
+}
+
+func (g *Gui) loaderTick() {
+	go func() {
+		for range time.Tick(time.Millisecond * 50) {
+			for _, view := range g.Views() {
+				if view.HasLoader {
+					g.userEvents <- userEvent{func(g *Gui) error { return nil }}
+					break
+				}
+			}
+		}
+	}()
+}
+
+// IsUnknownView return true if the contents of an error is "unknown view"
+func IsUnknownView(err error) bool {
+	return err.Error() == "unknown view"
```

Comment:
I think we  should use `ErrUnknownView` when may be change error message int the future

```go
return err.Error() == ErrUnknownView.Error()
```